tests/unit_tests/tests.js

```diff
@@ -1,5 +1,12 @@
 var test = QUnit.test;
 
+// We extract require function from define.js, in order to avoid defining
+// new module for every test just because we need require.
+var require = null;
+define("extractRequire", function(exportRequire, exports, module) {
+	require = exportRequire;
+});
+
```

Comment:
Couldn't you wrap all tests with this define instead of exporting require in this hacky way?